### ISSUE TITLE
[BUG] Weird Shader Issue

### ISSUE THREAD

**Issue by MrRedstoneA1** [TheCyberBrick/The-Betweenlands#1279]
**Betweenlands version:**

3.7.3 snapshot

**Forge version:**

14.23.5.2855

Singleplayer

**Description of the problem**

There's a way to use shaders alongside The Betweenlands. It was working fine until I got to the Primordial Malevolence. The fight progressed normally with no issue, but then the sky went black. The game was still running but the screen was frozen. I suspect it happens because heavy fog started. It happened again randomly after defeating the boss but did not happen before heavy fog.
There is no crash report but log shows some weird shader errors. I've included the log below. I had to shorten the log quite a bit.

**How to reproduce:**

1. Install The Betweenlands.
2. Install Optifine.
3. Install modified shaders that work with The Betweenlands.
4. Enter the marsh and wait for the heavy fog to appear or fight the Primordial Malevolence. Not sure which one causes it.
**Expected result:**

The game should continue to run normally.

**Actual result:**

Screen suddenly goes black and freezes.

**Log**
https://pastebin.com/BVT31sVp

**Comment by MrRedstoneA1** [comment]
I finally gave up trying to fix the issues. I turned off BL shaders and found a Sludgeon. The beam origin reflection caused a visual glitch I never could have imagined. Just to be clear, the Optifine shaders should be completely **disabled** in the dimension due to the slight modification, yet it feels like they're somehow leaking into the Betweenlands and interfering with the BL shaders. I cannot tell whether this issue is due to Optifine, the Betweenlands or both.
